### PR TITLE
Workflow deric

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: members/Deric.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Deric Khoo
+
```
Comment (NUSe0032202) [inline members/Deric.md:2]: Dummy comment

File: README.md [hunk 2]
```diff
@@ -3,3 +3,4 @@ This is a team repo for members of the team {Team ID}
```
Comment (NUSe0032202) [inline README.md:3]: Dummy Comment

**Review by NUSe0032202 — COMMENTED** [review]
Dummy Comment

File: README.md (hunk 2 above)
Comment (aver1493) [inline README.md:3]: Dummy comment

**Pull request merged**